src/database/seeds/BigQuestionsSeeder.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+
+class BigQuestionsSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('big_questions')->insert(
+            [
+                // listのデータ
+                ['name' => '東京の難読地名クイズ',
+                ],
```

Comment:
フォーマットだけ凄く気になる。。。？
エディタのフォーマットがかかるとこうなってしまう感じですか？
できれサジェストの感じが好ましいような気がしています (phpcsfixerというフォーマッターだと怒られるので)
```suggestion
                [
                    'name' => '東京の難読地名クイズ',
                ]
```